internal/command/command.go

```diff
@@ -87,6 +108,13 @@ func (c Command) execute(ctx context.Context, config *Config, args *Args, previo
 		return nil, err
 	}
 
+	cmdStr, err := c.ToString(config, args, outputs)
```

Comment:
Avoid ToString as a method name in Go. Either it's a Stringer or the method is doing more than string conversion.

Reply:
will get it in a follow up